### PR TITLE
Display tags in ExecutionView

### PR DESCRIPTION
**Pull request by kgevorkyan**
### What's done:
* Display tags in Execution view
* Add new column in the `Test` table
* Update corresponding tests

### CODE REVIEW

File: save-cloud-common/src/commonMain/kotlin/org/cqfn/save/test/TestDto.kt [hunk 1]
```diff
@@ -11,13 +11,15 @@ import kotlinx.serialization.Serializable
...
  * @property pluginName name of a plugin which this test belongs to
+ * @property tags list of tags of current test
  */
 @Serializable
 data class TestDto(
     val filePath: String,
     val pluginName: String,
     val testSuiteId: Long,
     val hash: String,
+    val tags: String?,
```
Comment (petertrr) [inline save-cloud-common/src/commonMain/kotlin/org/cqfn/save/test/TestDto.kt:22]: Maybe, we can store tags as lists in DTO and split when in `toDto` methods in entities?

**Review by petertrr — APPROVED** [review]

File: save-cloud-common/src/commonMain/kotlin/org/cqfn/save/agent/TestExecutionDto.kt [hunk 2]
```diff
@@ -22,4 +23,5 @@ data class TestExecutionDto(
     val startTimeSeconds: Long?,
     val endTimeSeconds: Long?,
     val testSuiteName: String? = null,
+    val tags: List<String>? = null,
```
Comment (petertrr) [inline save-cloud-common/src/commonMain/kotlin/org/cqfn/save/agent/TestExecutionDto.kt:26]:
```suggestion
    val tags: List<String> = emptyList(),
```
One of the reasons I suggested collection is that you can make it non-nullable

File: save-cloud-common/src/commonMain/kotlin/org/cqfn/save/test/TestDto.kt [hunk 3]
```diff
@@ -11,13 +11,15 @@ import kotlinx.serialization.Serializable
...
  * @property pluginName name of a plugin which this test belongs to
+ * @property tags list of tags of current test
  */
 @Serializable
 data class TestDto(
     val filePath: String,
     val pluginName: String,
     val testSuiteId: Long,
     val hash: String,
+    val tags: List<String>?,
```
Comment (petertrr) [inline save-cloud-common/src/commonMain/kotlin/org/cqfn/save/test/TestDto.kt:22]:
```suggestion
    val tags: List<String> = emptyList(),
```

**Review by petertrr — APPROVED** [review]

File: save-cloud-common/src/commonMain/kotlin/org/cqfn/save/agent/TestExecutionDto.kt (hunk 2 above)
Comment (kgevorkyan) [reply save-cloud-common/src/commonMain/kotlin/org/cqfn/save/agent/TestExecutionDto.kt]: Yes, good advise, fixed

**Pull request merged**